kalite/distributed/features/steps/superuser_create.py

```diff
@@ -46,6 +52,8 @@ def step_impl(context):
 
 @then("the modal won't dismiss")
 def step_impl(context):
+    driver = webdriver.Firefox()
+    driver.implicitly_wait(2)
```

Comment:
`implicitly_wait` is not a wait. It sets a timeout for other actions, and it only needs to be called once per WebDriver instance. See [the docs](http://selenium-python.readthedocs.org/api.html#selenium.webdriver.remote.webdriver.WebDriver.implicitly_wait)! The upshot is don't use it here if you expect it to wait 2 seconds (it won't, and that kind of behavior is awfully susceptible to race conditions), but instead use behave's [environmental controls](https://pythonhosted.org/behave/tutorial.html#environmental-controls) to set it once for the whole feature/test suite.